sys-apps/bleachbit/bleachbit-3.0.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
+PYTHON_REQ_USE="sqlite(+)"
+
+inherit desktop distutils-r1
+
+DESCRIPTION="Clean junk to free disk space and to maintain privacy"
+HOMEPAGE="https://www.bleachbit.org"
+SRC_URI="https://download.bleachbit.org/${P}.tar.bz2"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+gtk"
+
+RDEPEND="
+	dev-python/chardet[$PYTHON_USEDEP]
+	dev-python/scandir[$PYTHON_USEDEP]
+	gtk? ( dev-python/pygobject:3[$PYTHON_USEDEP] )
```

Comment:
https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Policies#USE-Controlled_Optional_RDEPENDs
You should use `optfeature` or similar instead.

Reply:
Virtually anyone would want a GUI for this. If it must not be controlled by a USE flag, my best bet would probably be to turn it into a hard dependency.